### PR TITLE
Volunteer UI Improvements

### PR DESCRIPTION
**Pull request by jellybob**
Most notably I've brought the schedule view in line with the rest of the site, and I've added the start of support for markdownified role descriptions to be displayed when looking at details.

### CODE REVIEW

File: static/js/volunteer-schedule.js [hunk 1]
```diff
@@ -74,36 +80,6 @@ function init_volunteer_schedule(data, all_roles, active_day) {
...
-            needed;
-
-        if (shift.min_needed === shift.max_needed) {
-            needed = shift.min_needed;
-        } else {
-            needed = shift.min_needed + ' - ' + shift.max_needed;
-        }
-
-        if (shift.is_user_shift) {
-            res.unshift(make_ele('p', 'You are currently signed up to this shift'));
```
Comment (Jonty) [inline static/js/volunteer-schedule.js]: Is this functionality no longer needed?

**Comment by Jonty** [comment]
LGTM, other than not being sure about the JS being removed - do we no longer need that functionality?

(Disclaimer: I've not had time to run this yet)

File: static/js/volunteer-schedule.js (hunk 1 above)
Comment (jellybob) [reply static/js/volunteer-schedule.js]: The same functionality is provided by the sign up buttons, which make it clearer in the list view that you're signed up for a given shift. Volunteer counts are also displayed there.

**Comment by jellybob** [comment]
The same functionality is provided by the sign up buttons, which make it clearer in the list view that you're signed up for a given shift. Volunteer counts are also displayed there.

**Comment by Jonty** [comment]
Merged and deployed

**Pull request merged**